abc_with_bb_support/src/opt/rwr/rwrUtil.c

```diff
@@ -27,7 +27,7 @@
 // precomputed data
 #ifdef _MSC_VER
 //MSVC requires that the size be known at declaration time if the variable
-//is static, so delcare it as non-static instead
+//is static, so declare it as non-static instead

```

Comment:
ABC is developed externally, so we don't accept modifications to it within the VTR source tree.